server/opts.go

```diff
@@ -1246,10 +1246,11 @@ func setClusterPermissions(opts *ClusterOpts, perms *Permissions) {
 // Temp structures to hold account import and export defintions since they need
 // to be processed after being parsed.
 type export struct {
-	acc  *Account
-	sub  string
-	accs []string
-	rt   ServiceRespType
+	acc     *Account
+	sub     string
+	accs    []string
+	rt      ServiceRespType
+	latency serviceLatency
```

Comment:
Let's make it a * since most of the time it may not be present. Also short names so maybe lat.